XamarinCommunityToolkitSample/Pages/Behaviors/EmailValidationBehaviorPage.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:behaviors="clr-namespace:XamarinCommunityToolkit.Behaviors;assembly=XamarinCommunityToolkit"
+             x:Class="XamarinCommunityToolkitSample.Pages.Behaviors.EmailValidationBehaviorPage">
+
+    <StackLayout Padding="{StaticResource ContentPadding}"
+                 Spacing="50"
+                 VerticalOptions="CenterAndExpand">
+        <Label Text="The text color will be changed to red in case of the wrong email is entered" />
+        <Entry Placeholder="Email">
+            <Entry.Behaviors>
+                <behaviors:EmailValidationBehavior InvalidStyle="{StaticResource InvalidEntryStyle}"/>
```

Comment:
I feel that it might be more clear for people, when they start looking at samples on how to use this, when the style is directly on this page? Instead of them also having to go look for where this style is defined.

IMO the samples should be as self-contained as possible